go/multipooler/manager/manager.go

```diff
@@ -1348,7 +1349,7 @@ func (pm *MultiPoolerManager) PrimaryStatus(ctx context.Context) (*multipoolerma
 		pm.logger.Error("Error iterating pg_stat_replication rows", "error", err)
 		return nil, mterrors.Wrap(err, "failed to read connected followers")
 	}
-	status.Followers = followers
+	status.ConnectedFollowers = followers
```

Comment:
This returns only followers that are currently connected. Open for discussion if we want to make it match what I did in GetFollowers, which returns the full list of configured standbys (whether they are currently connected or not).

Leaning towards revisiting at a later point when start using these API's in the consensus work.